### ISSUE TITLE
Test filter starting with dash is treated as an argument

### ISSUE THREAD

**Issue by caseyhoward** [denoland/deno#6975]
Any filter starting with a dash is treated as an argument to the deno command and not for the actual filter option.
To reproduce, run:

```sh
deno test --filter "- my filter"
```

And you will receive the following error:
```
error: Found argument '- ' which wasn't expected, or isn't valid in this context
```

Removing the dash works as expected:

```sh
deno test --filter "my filter"
```

 I would expect putting a dash in to work the same.

**Comment by JayHelton** [comment]
Ill grab this issue.

**Comment by bartlomieju** [comment]
Closed in #6975
